packages/react-native-razorpay/src/withReactNativeRazorpay.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  ConfigPlugin,
+  withInfoPlist,
+  createRunOncePlugin,
+} from "@expo/config-plugins";
+
+let pkg: { name: string; version?: string } = {
+  name: "react-native-razorpay",
+};
+
+try {
+  pkg = require("react-native-razorpay/package.json");
+} catch {
+  // empty catch block
+}
+
+type TPaymentApps = "tez" | "phonepe" | "paytmmp";
+
+/**
+ * Add UPI Intent apps on iOS
+ */
+const withReactNativeRazorpay: ConfigPlugin<{ paymentApps: TPaymentApps[] }> = (
+  config,
+  { paymentApps }
+) => {
+  return withInfoPlist(config, (cfg) => {
+    if (!paymentApps.length) {
+      cfg.modResults.LSApplicationQueriesSchemes = [];
```

Comment:
I was thinking that maybe instead of passing an empty array (which really doesn't make much sense by itself), we could pass every possible app by default so it comes prepackaged just like Android does, that way user doesn't have to interact with the config plugin beyond just adding plugin name to `app.json`

```suggestion
      cfg.modResults.LSApplicationQueriesSchemes = ["tez", "phonepe", "paytmmp"];
```

Reply:
This shouldn't return an empty array because it would nuke the modifications made by other plugins.